tests/test_occurrences_client.py

```diff
@@ -134,6 +136,17 @@ async def test_occurrences_without_victims(occurrences_client_and_get_mock):
     )
 
 
+@mark.asyncio
+async def test_occurrences_with_format_parameter(occurrences_client_and_get_mock):
+    client, mock = occurrences_client_and_get_mock
+    occurrences = Occurrences(client, id_state=42, type_occurrence="withVictim")
+    await occurrences()
+    mock.assert_called_once_with(
+        "http://127.0.0.1/api/v2/occurrences?idState=42&typeOccurrence=withVictim&page=1",
+        headers={"Authorization": "Bearer 42"},
+        format="None"
```

Comment:
Why a string `"None"` instead of the `None` value?

Reply:
I put as any value, just to confirm it is passing to `get` method.
A will change to a vlid value, like "df".

Reply:
But why do expect `"None"` if:
* you haven't passed `"None"` to `Occurrences`
* The default value for `format` is `None` (not `"None"`)

Reply:
> But why do expect `"None"` if:
> 
> * you haven't passed `"None"` to `Occurrences`
> * The default value for `format` is `None` (not `"None"`)

You are right. I understood your point and not only updated the expected value but also the `occurrences` call with the format parameter.